src/react-fontawesome-config.js

```diff
@@ -0,0 +1,11 @@
+import { library } from "@fortawesome/fontawesome-svg-core";
+
+import { faCartShopping, faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+
+import { } from "@fortawesome/free-regular-svg-icons";
```

Comment:
why is this need it? i think it's better not use this library.

Reply:
It's a config file where you only import icons you'll use and make them available for whole application. However I would consider replacing it with a smaller library based how many icons I will need